resources/dicts/patrols/beach/training/leaf-fall.json

```diff
@@ -1,3 +1,31 @@
 [
+    {
+        "patrol_id": "bch_train_lore1",
+        "biome": "beach",
+        "season": "leaf-fall",
+        "tags": ["training", "platonic", "trust"],
+        "intro_text": "p_l takes the patrol down to the beach to train, but, wait, something's odd here...",
+        "decline_text": "It's best to go back to camp and ignore the oddities on the beach.",
+        "chance_of_success": 30,
+        "exp": 5,
+        "success_text": [
+                "r_c explains that the sea has always given mysterious gifts to c_n, and that those strange ginger cat statues are probably just another gift from the ocean, probably.",
+                "The strange ginger cat statues seem to be purring slightly. r_c suggests they leave the beach as soon as possible, those things don't seem... Entirely of this world.",
+                "s_c laughs! The ginger cat statues are twoleg objects, they spin a tale almost unbelievable, but they swear it's true! The ginger cat statues used to live on something called a \"boat\".",
+                "Those ginger cat statues? That's just Garfield. s_c explains that, for what ever reason, twolegs worship the ginger cat so much they made statues they threw into the ocean."
```

Comment:
I think it may be stretching immersion for the cats to know Garfield's name.  I suggest one of the following replacements:

"That's just some story-character"
"That's just some twoleg deity" (bonus of being funny)
"s_c once heard a kittypet call him 'Garfield'."

Or, removing the "That's just Garfeild" sentence.

Reply:
Also `<i></i>` works for italics so you can use that instead of //

Reply:
\\ was used to keep the "" from ending the string, it wasn't for italics.